### PR TITLE
New trending plot of GTB Links decoding errors

### PR DESCRIPTION
**Pull request by selindra**
This plot shows how the amount of GTBLinks decoding errors was changing throughout the time. Was based on ITSqcTrendingFEE graph.

### CODE REVIEW

**Review by IsakovAD — COMMENTED** [review]
Could you please apply clang also to the CMake.txt file as well? I guess, one of checks fails because of that

File: Modules/ITS/src/ReductorBinContent.cxx [hunk 1]
```diff
@@ -65,6 +65,7 @@ void ReductorBinContent::update(TObject* obj)
     int numberOfBinsY = histo->GetYaxis()->GetNbins();
 
     for (int j = 1; j <= numberOfBinsY; j++) {
+      std::cout << "at bin : " << j << " filled by : " << histo->Integral(1, numberOfBinsX, j, j) << std::endl;
```
Comment (IsakovAD) [inline Modules/ITS/src/ReductorBinContent.cxx:68]: Remove this cout

File: Modules/ITS/src/TrendingTaskITSError.cxx [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+  // Preparing data structure of TTree
+  mTrend = std::make_unique<TTree>();
+  mTrend->SetName(PostProcessingInterface::getName().c_str());
+  mTrend->Branch("runNumber", &mMetaData.runNumber);
+  mTrend->Branch("Time", &mTime);
+  mTrend->Branch("Entries", &nEntries);
+
+  for (const auto& source : mConfig.dataSources) {
+    std::unique_ptr<ReductorBinContent> reductor(root_class_factory::create<ReductorBinContent>(source.moduleName, source.reductorName));
+    reductor->setParams(22, 0);
```
Comment (IsakovAD) [inline Modules/ITS/src/TrendingTaskITSError.cxx:65]: Could you define "22" as some "const int" variable in the header? 

This 22 corresponds to the number of error plots provided by your code

**Comment by IsakovAD** [comment]
@selindra Thank you for PR.

 Let`s see how the situation with build errors evolve after solving my comments

**Comment by IsakovAD** [comment]
@Barthelemy @knopers8 

Could you please add your approval to this PR? We need to have one more review, but Ivan is out of office for this week

**Review by knopers8 — APPROVED** [review]

**Comment by knopers8** [comment]
Same as with the other PR, it is waiting for Ivan's review specifically. I will force-merge as soon as the macOS check goes green.

**Pull request merged**